components/02-elements/swatch/swatch.config.js

```diff
@@ -34,30 +35,26 @@ module.exports = {
   },
   variants: [
     {
-      name: 'color',
+      name: 'attribute',
       context: {
         class: '',
-        title: 'Choose color',
+        title: 'Color',
         options: [
           {
-            class: 'swatch__option--color',
-            attributes: 'style="background: #000"',
-            text: ''
+            class: 'swatch__option--attributes',
+            attributes: 'style="background: url(../../images/swatch/swatch-orange.jpg); background-size: cover; background-position: center center;"'
           },
           {
-            class: 'swatch__option--color',
-            attributes: 'style="background: #fff"',
-            text: ''
+            class: 'swatch__option--attributes',
+            attributes: 'style="background: url(../../images/swatch/swatch-green.jpg); background-size: cover; background-position: center center;"'
           },
           {
-            class: 'swatch__option--color swatch__option--active',
-            attributes: 'style="background: #5cb85c"',
-            text: ''
+            class: 'swatch__option--attributes swatch__option--active',
+            attributes: 'style="background: url(../../images/swatch/swatch-red.jpg); background-size: cover; background-position: center center;"'
           },
           {
-            class: 'swatch__option--color',
-            attributes: 'style="background: #0275d8"',
-            text: ''
+            class: 'swatch__option--attributes',
+            attributes: 'style="background: url(../../images/swatch/swatch-white.jpg); background-size: cover; background-position: center center;"'
           }
         ]
       }
```

Comment:
In `components/02-elements/swatch/swatch.hbs` line 2, the `h5` is used... why? The heading shouldn't be relative to the design? maybe we should move it to config

Reply:
please add variant for category view

Reply:
ok, `h5` removed. And as we discussed on chat we don't need variant for category view b/c it's the same

Reply:
👍